core/templates/core/about.html

```diff
@@ -12,31 +12,28 @@
         <div class="container">
             <div class="static">
                 <h1 class="header">About</h1>
-                <p aria-labels="about-list-1">GREAT.gov.uk is the UK government’s single destination for information on trade, investment, tourism and education provided by:</p>
+                <p>Great.gov.uk brings together international trade and investment information and services from across government. It is maintained by the <a href="https://www.gov.uk/government/organisations/department-for-international-trade" title="Department for International Trade - GOV.UK">Department for International Trade (DIT)</a>.</p>
+				<p>DIT is the government department responsible for promoting and financing international trade and investment, and championing free trade.</p>
+				<h2 class="header">What's on great.gov.uk</h2>
+				<p aria-labels="about-list-1">You can:</p>
                 <ul id="about-list-1">
-                    <li><a href="http://www.gov.uk/dit">Department for International Trade</a>
-                    </li>
-                    <li><a href="http://www.visitbritain.org/">Visit Britain</a>
-                    </li>
-                    <li><a href="https://study-uk.britishcouncil.org/">British Council</a>
-                    </li>
+                    <li><a href="/">read guidance</a> to help you export your product or service
+                    </li>
+                    <li><a href="/triage">get market statistics and guidance</a> to match your experience</li>
+                    <li><a href="https://find-a-buyer.export.great.gov.uk/">create a profile</a> to showcase your business to overseas buyers</li>
+					<li><a href="https://selling-online-overseas.export.great.gov.uk/">find an online marketplace</a> to sell your products overseas</li>
+					<li><a href="https://opportunities.export.great.gov.uk/">apply for opportunities</a> to sell your product or service overseas</li>
+					<li><a href="/get finance">get finance advice and support from UK Export Finance</a></li>
```

Comment:
missed out a dash here - should be '/get-finance'

Reply:
a django thing: please use 
```
{% url 'get-finance' %}
```

that hooks into django's url routers and prevents us accidentally adding invalid links to the app (ones that 404).

Reply:
OK on both counts